templates/network.epp

```diff
@@ -22,4 +23,10 @@ KeepConfiguration=yes
 <%= $key %>=<%= $value %>
 <% } -%>
 <% } -%>
+<% $routes.each |$route| { -%>
 
+[Route]
+<% $route.each |$key, $value| { -%>
+<%= $key %>=<%= $value %>
+<% } -%>
+<% } -%>
```

Comment:
should we add a newline between line 31 and 32? that ensures that the file terminates properly with \n and the `[Route]` blocks will have a newline between

Reply:
I think this needs to be:
```suggestion
<% } %>
```

to have a proper line ending

Reply:
its copied from above so the file ending keeps the same.
on my system after rollout there is a proper file ending